packages/app/src/app/pages/common/Navigation/index.tsx

```diff
@@ -32,16 +34,25 @@ import {
   Wrapper,
 } from './elements';
 
-function Navigation({ signals, store, title, searchNoInput }) {
-  const { isLoggedIn, isPatron, user } = store;
+interface Props {
+  title: string;
+  searchNoInput?: boolean;
+}
+
+function Navigation({ title, searchNoInput }: Props) {
```

Comment:
@SaraVieira This can  be an arrow function

```suggestion
const Navigation = ({ title, searchNoInput }: Props) => {
```